### PR TITLE
Fixing NPE #6281

### PR DESCRIPTION
**Pull request by mbhaskar**
Fixing #6281

### CODE REVIEW

File: sdk/cosmos/microsoft-azure-cosmos/src/main/java/com/azure/data/cosmos/CosmosStoredProcedureResponse.java [hunk 1]
```diff
@@ -19,9 +19,11 @@
         }
     }
 
-    CosmosStoredProcedureResponse(StoredProcedureResponse response, CosmosContainer cosmosContainer) {
+    CosmosStoredProcedureResponse(StoredProcedureResponse response, CosmosContainer cosmosContainer,
+                                  String storedProcedureId) {
         super(response);
         this.storedProcedureResponse = response;
+        storedProcedure = new CosmosStoredProcedure(storedProcedureId, cosmosContainer);
```
Comment (kushagraThapar) [inline sdk/cosmos/microsoft-azure-cosmos/src/main/java/com/azure/data/cosmos/CosmosStoredProcedureResponse.java]: nit -> this.storedProcedure

Comment (mbhaskar) [reply sdk/cosmos/microsoft-azure-cosmos/src/main/java/com/azure/data/cosmos/CosmosStoredProcedureResponse.java]: done

**Pull request merged**